x/supply/client/rest/query.go

```diff
@@ -63,6 +77,22 @@ func totalSupplyHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 }
 
 // HTTP request handler to query the supply of a single denom
+type totalDenomSupply struct { //nolint: deadcode unsued
+	Height int64  `json:"height"`
+	Result string `json:"result"`
```

Comment:
shouldn't this be an `sdk.Int`?

Reply:
https://github.com/cosmos/cosmos-sdk/blob/master/client/lcd/swagger-ui/swagger.yaml#L1911

Reply:
based it off this

Reply:
interesting... `Coin.Amount` returns a `string` on the schema